lib/beaker-answers/answers.rb

```diff
@@ -4,12 +4,13 @@ module BeakerAnswers
   class Answers
 
     DEFAULT_ANSWERS =  StringifyHash.new.merge({
+      :q_install                                     => 'y',
       :q_puppet_enterpriseconsole_auth_user_email    => 'admin@example.com',
       :q_puppet_enterpriseconsole_auth_password      => '~!@#$%^*-/ aZ',
       :q_puppet_enterpriseconsole_smtp_port          => 25,
       :q_puppet_enterpriseconsole_smtp_use_tls       => 'n',
       :q_verify_packages                             => 'y',
-      :q_puppetdb_password                           => '~!@#$%^*-/ aZ',
+      :q_puppetdb_database_password                  => "'~!@#$%^*-/ aZ'",
```

Comment:
Previous to this commit, we set this value in the [version30](https://github.com/puppetlabs/beaker-answers/blob/master/lib/beaker-answers/versions/version30.rb#L76) file that would wrap the default in quotes. That implementation effectively meant the `DEFAULT_ANSWERS` value was a bit misleading, as we actually set that value wrapped in single quotes. The change to add in the single quotes here also requires the change [here](https://github.com/tvpartytonight/beaker-answers/blob/BKR-763/lib/beaker-answers/versions/version30.rb#L76).